classes/log/MonographFileEmailLogEntry.php

```diff
@@ -7,32 +7,17 @@
  * Copyright (c) 2003-2021 John Willinsky
  * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
  *
- * @class MonographFileEmailLogEntry
+ * @enum MonographFileEmailLogEntry
  *
  * @ingroup log
  *
- * @see MonographFileEmailLogDAO
- *
- * @brief Describes an entry in the monograph file email log.
+ * @brief Enumeration for monograph file log event types.
  */
 
 namespace APP\log;
 
-use PKP\log\EmailLogEntry;
+use PKP\log\core\EmailLogEventType;
 
-class MonographFileEmailLogEntry extends EmailLogEntry
+enum MonographFileEmailLogEntry: int implements EmailLogEventType
```

Comment:
This should be `MonographFileEmailLogEventType`, not `MonographFileEmailLogEntry`.